README.md

```diff
@@ -37,17 +37,24 @@ Please refer to our [coding standards](docs/coding-standards.md) for code styles
 
 2. Create and activate a virtual environment:
 
-   Unix based systems: 
+   Unix based systems:
+
    ```sh
    virtualenv env
    source env/bin/activate
    ```
-   Windows: 
+
+   Windows:
+
    ```sh
    python -m venv env
-   source env/Scripts/activate
+   source env/bin/activate
```

Comment:
```suggestion
   source env/Scripts/activate
```

This would stay the same as the original - its the correct path. Only difference is Git Bash users do use `source` to set the source.

Reply:
Changed, thanks for checking